src/pipecat/utils/watchdog_queue.py

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2024–2025, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+import asyncio
+
+from pipecat.utils.watchdog_reseter import WatchdogReseter
+
+
+class WatchdogQueue(asyncio.Queue):
```

Comment:
Same here, I thought we would also need to override the `task_done` method, and reset the watchdog once the task is completed.

Reply:
Yes!